src/main/groovy/com/palantir/gradle/gitversion/GitVersionPlugin.groovy

```diff
@@ -44,8 +45,11 @@ class GitVersionPlugin implements Plugin<Project> {
     private String gitDesc(Project project) {
         Git git = gitRepo(project)
         try {
-            DescribeFirstParentCommand describe = new DescribeFirstParentCommand(git.getRepository())
-            String version = describe.call() ?: UNSPECIFIED_VERSION
+            // back-compat: the JGit "describe" command throws an exception in repositories with no commits, so call it
+            // first to preserve this behavior in cases where this call would fail but native "git" call does not.
+            new DescribeCommand(git.getRepository()).call()
+
+            String version = runGitCommand(project.rootDir, "describe", "--tags", "--first-parent") ?: UNSPECIFIED_VERSION
```

Comment:
pretty sure we didn't have to strip a line separator from jgit's output, but likely we need to here?